### ISSUE TITLE
Error while catching exception when ModelsToArrayTransformer gone wrong.

### ISSUE THREAD

**Issue by firestorm23** [sonata-project/SonataAdminBundle#4015]
I got error while attempting to load 'sonata_type_model' in method **ModelsToArrayTransformer::getIdentifierValues**, but instead of getting Exception thrown, it says:

```
Attempted to load class "ClassUtils" from namespace "Sonata\AdminBundle\Form\DataTransformer".
Did you forget a "use" statement for e.g.  "Symfony\Component\Security\Acl\Util\ClassUtils", "Symfony\Component\Security\Core\Util\ClassUtils" or "Doctrine\Common\Util\ClassUtils"?
```

That was weird and i checked the source
The problem is

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Form/DataTransformer/ModelsToArrayTransformer.php
_line 126_

**ClassUtils** defined there is looked in namespace **Sonata\AdminBundle\Form\DataTransformer**

and that's why this weird error occurrs instead of throwing proper exception

``` php

namespace Sonata\AdminBundle\Form\DataTransformer;

...

    private function getIdentifierValues($entity)
    {
        try {
            return $this->modelManager->getIdentifierValues($entity);
        } catch (\Exception $e) {
            throw new \InvalidArgumentException(sprintf('Unable to retrieve the identifier values for entity %s', ClassUtils::getClass($entity)), 0, $e);
        }
    }

```

**Comment by greg0ire** [comment]
Good catch and thanks for your bug report!

1. Please fill the issue template, it's important.
1. Would you kindly provide a PR to fix that?
2. > Did you forget a "use" statement for e.g.  "Symfony\Component\Security\Acl\Util\ClassUtils", "Symfony\Component\Security\Core\Util\ClassUtils" or "Doctrine\Common\Util\ClassUtils"?
Which one do you think it is?

**Comment by greg0ire** [comment]
ping @firestorm23

**Comment by firestorm23** [comment]
Do you have any special conventions about PR's ?

**Comment by greg0ire** [comment]
Yes, they are extensively documented in CONTRIBUTING.MD

**Comment by greg0ire** [comment]
@firestorm23 do you need any help with this ?

**Comment by ReactiveRaven** [comment]
`Doctrine\Common\Util\ClassUtils` appears to be correct. There are other instances of it being used in previous PRs

* #1747 Use ClassUtils::getClass instead of get_class 
* #1601 use ClassUtils::getClass() instead of get_class

**Comment by jordisala1991** [comment]
Fixed in #4265
